src/view/home.js

```diff
@@ -1,87 +1,105 @@
 (function(){
-  LifeGamification.home = {};
+  const welcome_message = `
+    <p>Welcome to Life Gamification!<br>
+    To add your first skill go to Edit skills.</p>`;
 
-  //Functions to both Home and Edit views.
+  class SkillView extends Abstract.View {
+    _addTitle() {
+      this._appendNewElement(
+        'a', {"class": "skill__level-number"}, this.model.level);
+      this._appendNewElement(
+        'a', {"class": "skill__level-text"}, "lvl");
+      this._appendNewElement('a', {"class": "skill__name"}, this.model.name);
+    }
 
-  LifeGamification.home.viewLevelAndExp = function (skill) {
-    const number = LifeGamification.skillsView.findIndex(function (element) {
-      return element === skill;
-    });
-    $(`.level${number}`).html(`${skill.level}`);
-    $(`.name${number}`).html(`${skill.name}`);
-    $(`.exp${number}`).html(`
-      ${skill.expInThisLevel}/${skill.expTillNextLevel}`);
+    _addExpTillNextLevel() {
+      this._appendNewElement(
+        'a', {"class": "skill_experience"},
+        `${this.model.expInThisLevel}/${this.model.expTillNextLevel}`,
+      );
+    }
 
-    let percent = Math.floor(
-      100 * skill.expInThisLevel / skill.expTillNextLevel);
-    $(`.fill${number}`).html(`${percent}%`);
-    $(`.fill${number}`).css('width', `${percent}%`);
-  }
+    render() {
```

Comment:
Move render below `addProgressBar` function and change order of launching functions to match it's order of appearence in code.